buildout.cfg

```diff
@@ -5,6 +5,7 @@ parts = interpreter test
 
 [versions]
 Persistence =
+persistent = >= 4.2.3
```

Comment:
setup.py does not match this. It seems to me like it probably should.

Reply:
I was planning to remove this change before the merge because it was only necessary to force this new versions as it was not used by Zope because of the issue now fixed in this PR.

I'll have a look if this change is actually needed and has to be ported to `setup.py`.